ui/src/components/AppDetailedView.js

```diff
@@ -63,7 +83,8 @@ function mapStateToProps (state) {
 
 const mapDispatchToProps = {
   getAppById,
-  toggleNavigationModal
+  toggleNavigationModal,
```

Comment:
I wonder if anything in `mapDispatchToProps` is required, as these functions should all be available as props through the import at the top of the file, ie:
```
import { getAppById, toggleNavigationModal, toggleConfirmationModal } from '../actions/actions-ui';
```

Reply:
`toggleConfirmSaveAppModal` does come from Redux.

https://github.com/aerogear/mobile-security-service/blob/b5a132092606f1f17d7673b2b5c669c23ed391d5/ui/src/actions/actions-ui.js#L56

Reply:
My mistake, misread `mapDispatchToProps` as `mapStateToProps` - need :coffee:  :laughing: